### PR TITLE
ID Repository for InterOpera AAS-Submodels

### PR DESCRIPTION
**Pull request by quickhdsdc**
28 AAS submodel templates are being developed in the [InterOpera ](https://interopera.de/teilmodellprojekte/)Project. We would like to store the properties and their temporary semanticID here in IDTA repository.

### CODE REVIEW

File: InterOpera/NameplateLaserEngravingMachine/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The properties are defined based on the existing EU directives, standards and common practices at enterprises so that a far-reaching acceptance can be achieved. The defined properties are included in the SMC “AssetSpecificProperties” of the IDTA existing IDTA Submodel “Digital Nameplate for Industrial Equipment” V2.0. Therefore, this submodel "Digital Nameplate Laser Engraving Machine" is an extension of the existing template for LEM. Only the LEM-specific properties are defined in this sub-namespace, which have not been included in the exsiting dictionaries, aka. ECLASS and IEC CDD.
+
+## Status: `Submitted`
+The submodel "Digital Nameplate Laser for Engraving Machine" is under development. Its specification will be released soon.
+
+## MaxLaserPower (Property)
+
+Maximum optical output power of laser <br/>
+https://admin-shell.io/id/InterOpera/NameplateLaserEngravingMachine/MaxLaserPower
```
Comment (sebbader-sap) [inline InterOpera/NameplateLaserEngravingMachine/README.md:16]: Please follow a pattern like `https://admin-shell.io/InterOpera/NameplateLaserEngravingMachine/<version>/<revision>/MaxLaserPower`

File: InterOpera/NameplateLaserEngravingMachine/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+## Contact
+
+This sub-namespace is proposed by [Dachuan Shi](mailto:dachuan.shi@ipa.fraunhofer.de), [Fraunhofer IPA](https://www.ipa.fraunhofer.de/en/expertise/digitools-for-manufacturing.html). Alternatively, [Olga Meyer](mailto:olga.meyer@ipa.fraunhofer.de) can also be contacted for questions
```
Comment (sebbader-sap) [inline InterOpera/NameplateLaserEngravingMachine/README.md:21]: Is there an official InterOpera contact? People tend to leave organisations sometimes :-)

File: InterOpera/TechnicalDataLaserEngravingMachine/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## AdditionalAxis (Property)
+
+rotation axis <br/>
+https://admin-shell.io/id/InterOpera/TechnicalDataLaserEngravingMachine/AdditionalAxis
+
+## BeamDiameter (Property)
+
+physical size of the beam perpendicular to the direction of beam propagation <br/>
+https://admin-shell.io/id/InterOpera/TechnicalDataLaserEngravingMachine/BeamDiameter
```
Comment (sebbader-sap) [inline InterOpera/TechnicalDataLaserEngravingMachine/README.md:21]: Please see my comment on the URI structure above.

**Review by sebbader-sap — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please see my comments. In addition, please think about adding `major version` -> `revision` information in the URIs and reflect this also in the folder structure.

**Review by quickhdsdc — COMMENTED** [review]
The comments have been addressed.

File: InterOpera/NameplateLaserEngravingMachine/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+The properties are defined based on the existing EU directives, standards and common practices at enterprises so that a far-reaching acceptance can be achieved. The defined properties are included in the SMC “AssetSpecificProperties” of the IDTA existing IDTA Submodel “Digital Nameplate for Industrial Equipment” V2.0. Therefore, this submodel "Digital Nameplate Laser Engraving Machine" is an extension of the existing template for LEM. Only the LEM-specific properties are defined in this sub-namespace, which have not been included in the exsiting dictionaries, aka. ECLASS and IEC CDD.
+
+## Status: `Submitted`
+The submodel "Digital Nameplate Laser for Engraving Machine" is under development. Its specification will be released soon.
+
+## MaxLaserPower (Property)
+
+Maximum optical output power of laser <br/>
+https://admin-shell.io/id/InterOpera/NameplateLaserEngravingMachine/1/0/MaxLaserPower
```
Comment (sebbader-sap) [inline InterOpera/NameplateLaserEngravingMachine/README.md:16]: Please remove the `id` path segment: https://admin-shell.io/id/InterOpera/NameplateLaserEngravingMachine/1/0/MaxLaserPower --> https://admin-shell.io/InterOpera/NameplateLaserEngravingMachine/1/0/MaxLaserPower

Also for the other URIs.

**Review by sebbader-sap — CHANGES_REQUESTED** [review]
Please see my comment about the "id" segment. Rest looks good!

**Review by sebbader-sap — COMMENTED** [review]
Thanks for the update! I hereby confirm that the proposal is formally correct. Next step to get the final approval:
I will take the PR and present it in the "IDTA WS AAS" on July 19th. 
As soon as we have that, I will approve and merge the PR.

**Comment by sebbader-sap** [comment]
The AAS WS has approved the proposed identifiers.

**Comment by quickhdsdc** [comment]
> The AAS WS has approved the proposed identifiers.

Thank you!

**Review by sebbader-sap — APPROVED** [review]

**Pull request merged**